airbyte-integrations/connectors/source-google-sheets/source_google_sheets/source.py

```diff
@@ -73,11 +73,12 @@ def check(self, logger: AirbyteLogger, config: json) -> AirbyteConnectionStatus:
         # Check for duplicate headers
         spreadsheet_metadata = Spreadsheet.parse_obj(spreadsheet)
         grid_sheets = Helpers.get_grid_sheets(spreadsheet_metadata)
+        header_row_offset = Helpers.get_header_row_offset(config)
```

Comment:
Need add some default value, to avoid braking changes for old users which haven't this field in config.